Cmdr/BuiltInCommands/help.lua

```diff
@@ -29,9 +39,17 @@ return {
 			end
 		else
 			local commands = context.Cmdr.Registry:GetCommands()
-
-			for _, cmd in pairs(commands) do
-				context:Reply(("%s - %s"):format(cmd.Name, cmd.Description))
+			table.sort(commands, function(a, b)
+				return a.Group < b.Group
+			end)
+			context:Reply(ARGUMENT_SHORTHANDS)
```

Comment:
This line feels like it's right in the middle of the group logic. It should be moved up above the `local commands =` line, and add a space below it.